TELCODOCS-2315: Document known issues Telco RAN (#100482)
openshift/openshift-docs: release_notes/ocp-4-20-release-notes.adoc

+
--
** In an {aws-short} private cluster, the load balancer becomes stuck in the `pending` state and reports the error: `Error syncing load balancer: failed to ensure load balancer: could not find any suitable subnets for creating the ELB`.
* Starting with OCP 4.20, there is a decrease in the default maximum open files soft limit for containers. As a consequence, end users may experience application failures. To work around this problem, increase the container runtimes (CRI-O) ulimit configuration. (link:https://issues.redhat.com/browse/OCPBUGS-62095[OCPBUGS-62095])

** In {gcp-short} and {azure-short} private clusters, the load balancer is provisioned with an external IP address, when it should not have an external IP address.
--
+
There is no supported workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-57440[OCPBUGS-57440])

> * Deleting and recreating test workloads with a BlueField-3 NIC causes clock jumps due to inconsistent PTP synchronization. This disrupts time synchronization in test workloads. Manually synchronize the system time after deleting and re-creating test workloads to mitigate clock jumps.(link:https://issues.redhat.com/browse/RHEL-93579[RHEL-93579])

Reviewer: ```suggestion
* Deleting and recreating test workloads with a BlueField-3 NIC causes clock jumps due to inconsistent PTP synchronization. This disrupts time synchronization in test workloads. The time synchronization stabilizes when the workloads are stable (link:https://issues.redhat.com/browse/RHEL-93579[RHEL-93579])
```

+
--
** In an {aws-short} private cluster, the load balancer becomes stuck in the `pending` state and reports the error: `Error syncing load balancer: failed to ensure load balancer: could not find any suitable subnets for creating the ELB`.
* Starting with OCP 4.20, there is a decrease in the default maximum open files soft limit for containers. As a consequence, end users may experience application failures. To work around this problem, increase the container runtimes (CRI-O) ulimit configuration. (link:https://issues.redhat.com/browse/OCPBUGS-62095[OCPBUGS-62095])

** In {gcp-short} and {azure-short} private clusters, the load balancer is provisioned with an external IP address, when it should not have an external IP address.
--
+
There is no supported workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-57440[OCPBUGS-57440])

> * Deleting and recreating test workloads with a BlueField-3 NIC causes clock jumps due to inconsistent PTP synchronization. This disrupts time synchronization in test workloads. The time synchronization stabilizes once the workloads are stable. (link:https://issues.redhat.com/browse/RHEL-93579[RHEL-93579])(link:https://issues.redhat.com/browse/RHEL-93579[RHEL-93579])

Author: Updated as suggested.

+
--
** In an {aws-short} private cluster, the load balancer becomes stuck in the `pending` state and reports the error: `Error syncing load balancer: failed to ensure load balancer: could not find any suitable subnets for creating the ELB`.
* Starting with OCP 4.20, there is a decrease in the default maximum open files soft limit for containers. As a consequence, end users may experience application failures. To work around this problem, increase the container runtimes (CRI-O) ulimit configuration. (link:https://issues.redhat.com/browse/OCPBUGS-62095[OCPBUGS-62095])

** In {gcp-short} and {azure-short} private clusters, the load balancer is provisioned with an external IP address, when it should not have an external IP address.
--
+
There is no supported workaround for this issue. (link:https://issues.redhat.com/browse/OCPBUGS-57440[OCPBUGS-57440])

* Deleting and recreating test workloads with a BlueField-3 NIC causes clock jumps due to inconsistent PTP synchronization. This disrupts time synchronization in test workloads. The time synchronization stabilizes when the workloads are stable. (link:https://issues.redhat.com/browse/RHEL-93579[RHEL-93579])

> * Event logs for GNR-D interfaces are ambiguous due to identical three-letter prefixes ("eno"). As a consequence, affected interfaces are not clearly identified during state changes. To work around this problem, modify event logging to include interface details, ensuring events now clearly indicate which interface is affected by state changes. (link:https://issues.redhat.com/browse/OCPBUGS-62817[OCPBUGS-62817])

Reviewer 2: Change: 
To work around this problem, modify event logging to include interface details, ensuring events now clearly indicate which interface is affected by state changes. (link:https://issues.redhat.com/browse/OCPBUGS-62817[OCPBUGS-62817])
To:
To work around this problem, rename interfaces used by ptp-operator to follow the "path" naming convention, ensuring per clock events are identified properly based on interface names and events now clearly indicate which clock is affected by state changes. (link:https://issues.redhat.com/browse/OCPBUGS-62817[OCPBUGS-62817])

Original release note for reference:
> If the user is configuring eno* interfaces with ptp-operator, they are not accurately identified when sending event. This is because the format of the eno interfaces is determined by the bios and does not allow identifying ports belonging to the same nic.
PHC clocks identified by an interface alias in eno nics will all show up as "enox" in events
The user should force interface naming to follow the "path" naming convention in the kernel so that eno interface are renamed to follow intel or mellanox naming convention such as ens1f0. This can be achieved by modifying the siteconfig to create a udev rule.

Author: @edcdavid, thank you for the review! I've made your requested changes. Could you please review and confirm if they're correct? Please add an `/lgtm` comment to the PR if no changes are required. Thank you!

Reviewer 2: looks good, just reading it again, maybe we could add a reference to the "path" scheme at https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/configuring_and_managing_networking/consistent-network-interface-device-naming_configuring-and-managing-networking#network-interface-naming-policies_consistent-network-interface-device-naming
<img width="871" height="103" alt="image" src="https://github.com/user-attachments/assets/3250871c-e8d0-4f7a-9511-d4502c48b1d3" />

Author: Updated as suggested.

+
--
** In an {aws-short} private cluster, the load balancer becomes stuck in the `pending` state and reports the error: `Error syncing load balancer: failed to ensure load balancer: could not find any suitable subnets for creating the ELB`.

> * Starting with OCP 4.20, there is a decrease in the default maximum open files soft limit for containers. As a consequence, end users may experience application failures. To work around this problem, increase the container runtimes (CRI-O) ulimit configuration. (link:https://issues.redhat.com/browse/OCPBUGS-62095[OCPBUGS-62095])

Reviewer 3: @kdrake20 Is this note in the correct place? Looking at it, I'm not sure how it relates to the bullet points either side of it or the bug that covers the overall bulleted list
